data/human/human missions.txt

```diff
@@ -5416,3 +5416,99 @@ mission "Deal Change 2"
+			`	You thank the clerk and take the package back to your ship.`
+			choice
+				`	(Take a picture of the incorrect address.)`
+				`	(Don't bother taking a picture.)`
+					accept
+			action
+				set "took picture of quicksilver package"
+			`	You take a few photographs of the address label, just to be sure that you've got a clear picture of the wrong address.`
+				accept
+	on complete
+		conversation
+			`When you talk to a post officer in Quicksilver's spaceport and point out the incorrect address before handing it over, you note that he exhibits the same frustration the clerk on Silver did regarding the frequent switch-ups. He takes the package, muttering something about offworlders "not knowing the difference even if it smacked 'em in the face" as he did.`
+			`	Once you return to your ship, you notice that the man who gave you that package had sent you a message. Or, to be more precise, a series of messages. An angry series of messages.`
+			`	"WHAT TOOK YOU SO LONG???" one of them said. "I NEEDED THAT DELIVERED DAYS AGO", says the next one.`
+			branch picture "didn't take picture"
+				has "took picture of quicksilver package"
+
+			label picture
+			branch "picture success" "picture failure"
+				random < 50
+
+			label "picture failure"
+			`	You send the man the picture you took of his error and point out that it was his mistake that got the package initially got sent to the wrong place, and politely inform him that had he wanted a smoother delivery, then maybe he should've filed a proper job listing and made sure that he place the correct address.`
+			`	He leaves you on read.`
+
+			label "picture success"
+			action
+				payment 22100
+			`	You send the man the picture you took of his error and point out that it was his mistake that got the package initially got sent to the wrong place, and politely inform him that had he wanted a smoother delivery, then maybe he should've filed a proper job listing and made sure that he place the correct address.`
+			`	He sees the message, but doesn't send a reply. Just before you walk away from your ship's terminal, however, you see a new message appear:`
+			`	"I am so sorry."`
+			`	The man then proceeds to apologize profusely for his mistake and his outburst, offering to pay an extra 5000 credits for the trouble. You gladly accept.`
+
+			label "didn't take picture"
+			`	You try to defend yourself, but the man remains adamant that it was your mistake that got the package sent so late. After a frustrating back-and-forth, you decide to relent and (reluctantly) offer an apology. He ignores it and threatens to get spaceport authorities to "STOP OFFERING JOBS TO INCOMPETENTS LIKE YOU."`
+			`	You don't bother to reply.`
```

Comment:
```suggestion
			label picture
			`	You send the man the picture you took of his error and point out that it was his mistake that got the package initially got sent to the wrong place, and politely inform him that had he wanted a smoother delivery, then maybe he should've filed a proper job listing and made sure that he place the correct address.`
			branch "picture success" "picture failure"
				random < 50

			label "picture failure"
			`	He leaves you on read.`
				goto end

			label "picture success"
			action
				payment 22100
			`	He sees the message, but doesn't send a reply. Just before you walk away from your ship's terminal, however, you see a new message appear:`
			`	"I am so sorry."`
			`	The man then proceeds to apologize profusely for his mistake and his outburst, offering to pay an extra 5000 credits for the trouble. You gladly accept.`
				goto end

			label "didn't take picture"
			`	You try to defend yourself, but the man remains adamant that it was your mistake that got the package sent so late. After a frustrating back-and-forth, you decide to relent and (reluctantly) offer an apology. He ignores it and threatens to get spaceport authorities to "STOP OFFERING JOBS TO INCOMPETENTS LIKE YOU."`
			`	You don't bother to reply.`
			
			label end
```

Reply:
Can share the picture comment before it branches, then need an end label to jump to after each.